src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -27,10 +27,24 @@
 
 #include "tusb_option.h"
 
-#if TUSB_OPT_DEVICE_ENABLED && CFG_TUSB_MCU == OPT_MCU_STM32F4
+#if TUSB_OPT_DEVICE_ENABLED && ( CFG_TUSB_MCU == OPT_MCU_STM32F4 || \
+                                 CFG_TUSB_MCU == OPT_MCU_STM32H7 )
```

Comment:
Can we add the supported F1 variants to this ifdef? We can say ((CFG_TUSB_MCU == OPT_MCU_STM32F1) && (defined(stm32f105xc) || defined(stm32f107xc))).

Reply:
This sounds like it'll work if you can test it beforehand. I am unable to test F1 boards because I don't have any :).

Reply:
Unfortunately I don't have a Synopsys F1...

(I only have the ST-USB F1). We can skip it you think it would add confusion.